Pod/Classes/Terminals/Utility/JsonDoc.m

```diff
@@ -81,6 +81,10 @@ - (BOOL) has:(NSString*)key
     return [_dict valueForKey:key] != nil;
 }
 
+- (NSString *) getValueAsString:(NSString *)key{
+    return [NSString stringWithFormat:@"%@", [self getValue:key]];
```

Comment:
Might want to allow for `[self getValue:key]` to be nullable here.
cc @lundhjem

Reply:
Good catch! Done.